clifford/cga.py

```diff
@@ -165,10 +165,8 @@ def __init__(self,cga, *args):
         # from vectors on flat         
         else:
             nulls = list(map(self.cga.null_vector,args))
-            if self.einf not in nulls:
-                nulls = list(nulls)+[self.einf]
-                
-            self.mv = reduce(op,nulls) 
+            self.mv = reduce(op,nulls)
+            self.mv = op(self.mv, self.einf) or self.einf
```

Comment:
can you explain more about this? i dont follow  it.

Reply:
Oops. I made my own silly mistake. It should be `op(self.mv, self.einf) or self.mv`. If a multiple of einf is in the list of blades, it will outer to zero, and we shouldn’t multiply in that additional einf factor.

Reply:
why not